clients/githubrepo/client.go

```diff
@@ -259,8 +261,33 @@ func CreateGithubRepoClientWithTransport(ctx context.Context, rt http.RoundTripp
 	httpClient := &http.Client{
 		Transport: rt,
 	}
-	client := github.NewClient(httpClient)
-	graphClient := githubv4.NewClient(httpClient)
+
```

Comment:
Can you please include tests for these?

Reply:
@naveensrinivasan , can you give me some pointers / help on where to get started? (not a Go programmer, so testing in Go is new to me). 

I see a generic setup for creating the clients during the tests here: [clients/githubrepo/githubrepo_suite_test.go](https://github.com/rajbos/scorecard-ghes/blob/b66365206d05232fe1b37ae24acbadc01733723f/clients/githubrepo/githubrepo_suite_test.go#L65), and another generic mock here [clients/mockclients/repo_client.go](https://github.com/rajbos/scorecard-ghes/blob/ghes-support/clients/mockclients/repo_client.go).